tests/cypress/e2e/sar/sar.cy.js

```diff
@@ -0,0 +1,146 @@
+// Feature: MCPAR E2E Form Submission
```

Comment:
typo: MCPAR

Reply:
Thanks @gmrabian after speaking with Jess we decided to just push what was done so far. There's possible conversations about reworking cypress or even finding a new solution. After attempting doing the full form, utilizing the wp/form.cy.js still needs a good amount of extra logic to work with SAR. I think it might be better confirm if we're doing a SPIKE to evaluate the e2e tooling before finishing this.